application/Views/errors/html/error_exception.php

```diff
@@ -86,12 +86,15 @@
 									<div class="args" id="<?= $args_id ?>">
 										<table cellspacing="0">
 
-										<?php foreach ($row['args'] as $key => $value) : ?>
-											<?php
-												$mirror = isset($row['class']) ? new \ReflectionMethod($row['class'], $row['function']) :
-													new \ReflectionFunction($row['function']);
-												$params = $mirror->getParameters();
-											?>
+										<?php
+										$params = null;
+										// Reflection by name is not available for closure function
+										if( substr( $row['function'], -1 ) !== '}' )
```

Comment:
I'll merge for now, and we'll see if it causes a concern down the line. I think it should probably be fine. Thanks.